sdk/identity/Azure.Identity/src/AuthenticationResponse.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.Identity
+{
+    public class AuthenticationResponse
```

Comment:
Would AccessToken be a better name for this type? Also, should it be a struct?

Reply:
I didn't make this a struct because this is cached in AzureCredential, and I wanted to make sure that when the cached value is updated it's updated atomically, so readers can't read a torn state of the object.

As far as the name is concerned I'm open to changing it.  If I changed the type name to AccessToken would the AccessToken property just be called Value?  or maybe Token?